app/server/utils/auth.py

```diff
@@ -293,7 +293,8 @@ def get_user_organisations(user):
     organisations = dict(
         active_organisation_name=active_organisation.name,
         active_organisation_id=active_organisation.id,
-        organisations=[dict(id=org.id, name=org.name) for org in user.organisations]
+        active_organisation_token=active_organisation.token.symbol,
+        organisations=[dict(id=org.id, name=org.name, token=org.token.symbol) for org in user.organisations]
```

Comment:
fine until we refactor... see https://github.com/teamsempo/SempoBlockchain/pull/129#issuecomment-578474802